legend-engine-pure-code-compiled-core/src/main/resources/core/legend/service/metamodel.pure

```diff
@@ -59,6 +59,10 @@ Class meta::legend::service::metamodel::PureExecution extends Execution
    func : FunctionDefinition<Any>[1];
 }
 
+Class meta::legend::service::metamodel::PureInlineExecution extends PureExecution
```

Comment:
Not sure about how this is modelled.  

IMO an InlineExecution is not an execution like the SingleExecution or MultiExecution.

InlineExecution seems to be about how the mapping and runtime are defined in the service model. 
Single and MultiExecution are about how the execution can be choose different one/more runtimes based on a key.